.env.example

```diff
@@ -0,0 +1,159 @@
+# Pydantic AI DevContainer Environment Variables
+# Copy this file to .env and fill in your actual values
```

Comment:
Could our devcontainer setup do this automatically? I think there's also a way to open a file on launch, or even to show a form to fill in these env vars

Reply:
`initializeCommand` copies now an external `.env` file into the container on launch.